src/components/Link/__stories__/Link.new.stories.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import type {Meta, StoryFn} from '@storybook/react';
 
-import {Link} from '../Link';
+import {Link, type LinkProps} from '../Link';
```

Comment:
We're not using this TS syntax

Reply:
Well, then it should be regulated by the linter. The language provides me with tools, and I use them. In addition, this constuction is used in other components.
I can set up an appropriate rule to ban inline important. But at first glance it looks strange. Why did you decide not to use it?

Reply:
I remember the main reason was that it requires to use TS 4.5+